doc/rst/meta/modules/standard.rst

```diff
@@ -6,19 +6,111 @@ Standard Modules
 Standard modules are those which describe features that are considered
 part of the Chapel Standard Library.
 
+
+Automatic Modules
+-----------------
+
 All Chapel programs automatically ``use`` the modules :chpl:mod:`Builtins` ,
 :chpl:mod:`Math` , and :chpl:mod:`Types` by default.
 
+.. toctree::
+   :maxdepth: 1
+
+   Builtins <standard/Builtins>
+   Math <standard/Math>
+   Types <standard/Types>
+
+
+Data Structures
+---------------
+
+.. toctree::
+   :maxdepth: 1
+
+   Heap <standard/Heap>
+   List <standard/List>
+   Map <standard/Map>
+   Set <standard/Set>
+
+   
+Diagnostics
+-----------
+
+.. toctree::
+   :maxdepth: 1
+
+   CommDiagnostics <standard/CommDiagnostics>
+   Memory <standard/Memory>
+   Time <standard/Time>
+
+   
+Files/IO
+--------
+.. toctree::
+   :maxdepth: 1
+
+   FileSystem <standard/FileSystem>
+   IO <standard/IO>
+   Path <standard/Path>
+   Regexp <standard/Regexp>
+
+
+System/Interoperability
+-----------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   CPtr <standard/CPtr>
+   DateTime <standard/DateTime>
```

Comment:
In my mind `DateTime` is a general-purpose utility, not about "system".

Reply:
Moved to new Utilities section